crowbar_framework/app/assets/stylesheets/application.scss

```diff
@@ -4,6 +4,7 @@
 @import "codemirror";
 @import "bootstrap-sprockets";
 @import "bootstrap";
+@import "bootstrap/fileinput";
```

Comment:
Wrong file, move that to `crowbar_framework/app/assets/javascripts/bootstrap.js` please.